app/(tabs)/(home)/index.tsx

```diff
@@ -146,26 +236,74 @@ export default function HomeScreen() {
           </ButtonGroup>
         );
     }
-  }
+  };
+
+  const ShapeSource = ({ route }: { route: Route }) => {
+    geoJSON = JSON.parse(route.geoJson);
+    return (
+      <MapboxGL.ShapeSource id="route" shape={geoJSON}>
+        <MapboxGL.LineLayer
+          id="route"
+          belowLayerID="road-label"
+          style={{
+            lineColor: "#b8b8b8",
+            lineWidth: 5,
+            lineJoin: "round",
+          }}
+        />
+        <MapboxGL.CircleLayer
+          id="pointLayer"
+          filter={["==", "$type", "Point"]}
+          style={{
+            circleColor: "black",
+            circleRadius: 6,
+          }}
+        />
+      </MapboxGL.ShapeSource>
+    );
+  };
+
+  // observe the route (tracks updates to the route)
+  const enhance = withObservables(["route"], ({ route }: { route: Route }) => ({
+    route,
+  }));
+
+  const EnhancedShapeSource = enhance(ShapeSource);
+
+  // observe routes of a tour (only tracks create and delete in the routes table)
+  const Bridge = ({ routes }: { routes: Route[] }) => {
+    if (routes.length === 0) return null;
+    return <EnhancedShapeSource route={routes[0]} />;
```

Comment:
Also wenn ich mehrere Routen für eine Tour importiere, wird immer die erste genommen? Brauchen wir vielleicht eine Story für "Alle Routen in den Toureinstellungen anschauen und die aktive Route auswählen können"?

Reply:
Das nimmt sich die Route von einer Tour. Dabei kann eine Tour nur eine Route haben. Und es heißt nur routes weil die Query immer eine Liste ist und es keine 1:1 Beziehungen in WatermelonDB gibt.